R/generate_simulated_data.R

```diff
@@ -50,6 +50,15 @@
 #' LOD across sites
 #' @param input_params_path path to the toml file with the parameters to use
 #' to generate the simulated data
+#' #' @param corr_function Correlation function for spatial site correlations
+#' @param corr_fun_params Parameters required for correlation function
+#' @param phi_rt Coefficient for AR(1) temporal correlation on subpopulation
+#' deviations.
+#' @param sigma_eps Parameter for construction of covariance matrix of spatial
+#' epsilon.
+#' @param scaling_factor Scaling factor for aux site.
+#' @param aux_site_bool Boolean to use the aux site framework with
```

Comment:
Why would we want to default this to FALSE?

Reply:
I think we want to have the defaults here use the spatial component, and then handle the auxiliary site as you had written. And then we want to ensure that if we don't want the spatial component to be included, that the auxiliary site R(t) gets handled the way it currently does (treated as just another site)